### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by janosbinder**
Minor Changes:

* run 'go fmt' on cloudfoundry/cfapi/{app_manager.go, domain_manager.go, quota_manager.go, service_manager.go, resource_cf_org_quota.go}
* increased timeout for linters
* updated cli cf dependency to v6.29.0 (https://github.com/cloudfoundry/cli/tree/v6.29.0 - two weeks after the commit id in vendor, I could not check out the current commit id of https://github.com/cloudfoundry/cli/commit/3379b668c1061a67f3762366e4cda6388559f927) 
* removed PatchResource from gateway.go, which is a very bad idea to modify files in the vendor directory (introduced by https://github.com/mevansam/terraform-provider-cf/pull/125)
* added PatchResource logic to space_manager.go and service_manager.go

### CODE REVIEW

File: vendor/code.cloudfoundry.org/cli/cf/configuration/coreconfig/config_data.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ type Data struct {
 func NewData() *Data {
 	data := new(Data)
 
-	data.UAAOAuthClient = "cloudfoundry"
+	data.UAAOAuthClient = "cf"
```
Comment (gberche-orange) [inline vendor/code.cloudfoundry.org/cli/cf/configuration/coreconfig/config_data.go]: @janosbinder any idea of how this change impacts terraform-cloudfoundry-provider users ? 

The default in cf-deployment is still to use a "cf" uaa client, see https://github.com/cloudfoundry/cf-deployment/blob/a456c2805b7623a9c241294fc09d2a65479b6ea5/cf-deployment.yml#L531-L538

I don't understand how this could have worked before ? Maybe it's dead code.

Comment (janosbinder) [reply vendor/code.cloudfoundry.org/cli/cf/configuration/coreconfig/config_data.go]: I think it was renamed by a mistake in #44 . See the changes here https://github.com/mevansam/terraform-provider-cf/blob/0.9.9/vendor/code.cloudfoundry.org/cli/cf/configuration/coreconfig/config_data.go by selecting "blame".

In the previous commit it used to be "cf" as well, see here https://github.com/cloudfoundry/cli/blob/3379b668c1061a67f3762366e4cda6388559f927/cf/configuration/coreconfig/config_data.go#L49 The `govendor sync` restored the correct contents.

The reason, why we have not faced any issue, as we did not use this part of the code.

In summary, in the `vendor` folder I would just only change the `vendor.json` in the future. Let me know, if the team has anything against this idea.

**Comment by gberche-orange** [comment]
@psycofdj can you please add your review ?
